test/timezone/lookup/test_geonames.rb

```diff
@@ -35,6 +49,19 @@ def test_lookup
     assert_equal 'Australia/Adelaide', mine.lookup(*coordinates)
   end
 
+  def test_lookup_with_etc
+    %w(arctic atlantic).each do |point|
```

Comment:
Yes, you are right of course. I just copypasted test from the lookup method in rough-and-ready fashion, so did not made a proper changes for multiple points test. I'll change it.